docs/tutorial/getting-started.md

```diff
@@ -1,27 +1,32 @@
-# Getting Started
+# Deploy the jenkins-k8s charm for the first time
 
-## What you'll do
+## What we do
 
 - Deploy the [jenkins-k8s charm](https://charmhub.io/jenkins-k8s)
+- Access the UI
 - Deploy and integrate agents
-- Get admin password
 
 The jenkins-k8s charm helps deploy a Jenkins automation server application with ease and
 also helps operate the charm. This
-tutorial will walk you through each step of deployment to get a basic Jenkins server deployment.
+tutorial will walk through each step of deployment to get a basic Jenkins server deployment.
 
-### Prerequisites
+### Requirements
 
-To deploy jenkins-k8s charm, you will need a Juju bootstrapped with any Kubernetes controller.
-To see how to bootstrap your Juju installation with MicroK8s, please refer to the documentation
-on MicroK8s [installation](https://juju.is/docs/olm/microk8s).
+- A machine with amd64 architecture.
+- Juju 3 installed.
+- Juju MicroK8s controller created and active named microk8s. [MetalLB addon](https://microk8s.io/docs/addon-metallb) should be enabled for traefik-k8s to work.
+- LXD controller created and active named lxd (optional).
+- All the requirements can be met using the [Multipass charm-dev blueprint](https://juju.is/docs/juju/set-up--tear-down-your-test-environment#heading--set-up---tear-down-automatically). Use the Multipass VM shell to run all commands in this tutorial.
+
+For more information about how to install Juju, see [Get started with Juju](https://juju.is/docs/olm/get-started-with-juju).
 
 ### Set up the tutorial model
 
 To easily clean up the resources and to separate your workload from the contents of this tutorial,
-set up a new Juju model with the following command.
+set up a new Juju model in the microk8s controller with the following command.
```

Comment:
Nitpick: microk8s --> MicroK8s
or
microk8s --> `microk8s` (if you're referring to the name of the controller)